expat/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: expat
+  version: 2.1.0
+
+source:
+  fn: expat-2.1.0.tar.gz
+  url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz
+  md5: dd7dab7a5fea97d2a6a43f511449b7cd
+
+requirements:
+  build:
+    - gcc
```

Comment:
Nop, Python is not built with `gcc`. It's built with the gcc that comes in CentOS 5. We really use `gcc` for very few packages (the R ones mostly). So until now we've demanded `libgcc` as a run dep whenever we use `gcc` as a build dep.

But perhaps that's only necessary when building C++ libraries instead of C ones (because the C standard is much more stable).

@asmeurer, could you clarify when it's necessary to use `libgcc`?